common/server_config.py

```diff
@@ -31,7 +31,10 @@ class ServerConfig(BaseModel):
 
     # Amount of time clients will waiting for other clients to join before the
     # game is started with bots replacing any missing clients.
-    wait_time_before_bots_seconds: int = 30
+    waiting_time_before_bots_seconds: int = 30
+
+    # Path to the file where player scores are saved.
```

Comment:
I would call this `score_filename` or `high_score_filename` (to be consistent with the HighScore class) and make the path implicit.